.vscode/settings.json

```diff
@@ -11,6 +11,7 @@
     "editor.defaultFormatter": "charliermarsh.ruff",
     "[python]": {
         "editor.formatOnSave": true,
+        "editor.defaultFormatter": "ms-python.autopep8",
```

Comment:
this is likely different than our linter which is why our CI pipeline is failing. You can see how it is configured here: https://github.com/Kaggle/kagglehub/blob/e739f0692816be9ee9d21816e1cc8471b3bf9b16/pyproject.toml#L79-L83

Reply:
Can you re-format it using `hatch run lint:fmt` and let me know when it is done to make it easier to review.

Reply:
Done. Am still learning the differences of these formater. Will be more careful next time.